repositories/user_repository.js

```diff
@@ -1,35 +1,34 @@
 const { getKnex } = require("../db");
- 
+const knex = getKnex();
```

Comment:
this doesn't work for me

on GET /users:

`TypeError: knex is not a function
    at Object.getAllUsers (/Users/aliona/Desktop/workspace/Fun/CatBytes/webplatform-backend/repositories/user_repository.js:5:15)
    at Object.getAllUsers (/Users/aliona/Desktop/workspace/Fun/CatBytes/webplatform-backend/services/user_service.js:6:23)
    at /Users/aliona/Desktop/workspace/Fun/CatBytes/webplatform-backend/routes/users.js:58:41
    at Layer.handle [as handle_request] (/Users/aliona/Desktop/workspace/Fun/CatBytes/webplatform-backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/aliona/Desktop/workspace/Fun/CatBytes/webplatform-backend/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/aliona/Desktop/workspace/Fun/CatBytes/webplatform-backend/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/aliona/Desktop/workspace/Fun/CatBytes/webplatform-backend/node_modules/express/lib/router/layer.js:95:5)
    at /Users/aliona/Desktop/workspace/Fun/CatBytes/webplatform-backend/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/Users/aliona/Desktop/workspace/Fun/CatBytes/webplatform-backend/node_modules/express/lib/router/index.js:346:12)
    at next (/Users/aliona/Desktop/workspace/Fun/CatBytes/webplatform-backend/node_modules/express/lib/router/index.js:280:10)`